components/features/global/alert-bar/default.jsx

```diff
@@ -1,32 +1,27 @@
 /*  /components/features/article/alert-bar/default.jsx  */
 
-import React from 'react';
+import React, { Fragment } from 'react';
 import PropTypes from 'prop-types';
 import { useComponentContext } from 'fusion:context';
-import { useEditableContent } from 'fusion:content';
 
 const AlertBar = () => {
   /* retrieve custom fields from fusion's component context */
   const componentContext = useComponentContext();
   const {
     alertType,
     link,
+    linkText,
     text,
   } = componentContext.customFields;
 
-  /* get inline editing function from fusion's content */
-  const { editableField } = useEditableContent();
-
   return (
-    <section className={`alert alert-${alertType} w-100`} role='alert' >
-      <a
-        {...editableField('text')}
```

Comment:
why was this removed? editableFields were also covered in lab-06 so this solution should be here. Correct me if im wrong but I dont believe any changes are needed on this branch/code